### PR TITLE
Small-scale tech level rebalancing, syringe gun buff, syringegun tech levels nerf

### PR DESCRIPTION
**Pull request by Dorsisdwarf**
Syringe guns were seriously overpowered for research. They were basically the key to jump-starting tech, so sod that. R&D should take more time than it does.
In order to help prevent them from becoming unloved and alone like the cable-layer, I made them start with salbutamol as a known reagant so they can help people.

This also adds the ability to emag a mech syringe gun, allowing it to synthesize CR, formaldehyde, and neurotoxin (the dangerous one).

Also adds a bunch of tech to things people in OOC thought needed tech levels.

Also Also adds tech levels to a bunch of things that didn't have them (abductor gear etc)

### CODE REVIEW

**Comment by Supermichael777** [comment]
dose this fix the syringe guns scanning powers? I recall goon chem broke that.

**Comment by Dorsisdwarf** [comment]
I'm not aware of any syringe gun bugs?

**Comment by AlexCrimson** [comment]
Why the fuck would you do this? If people dont use that syringe gun, they will just find something else. Just making things more annoying. Good jawb i guess.

**Comment by Dorsisdwarf** [comment]
Welcome to balancing hope you had a nice day

**Comment by AlexCrimson** [comment]
Nice balancing. People will just find new items to use for the levels. If you want to balance R&D then you need to completely redesign the research system, not change a few levels that you can get from other items just as easily. Syringe Gun was used just because it was the item that took the least effort to get.

**Comment by Dorsisdwarf** [comment]
No, it was used because it let you jump 3 techs to the highest tier instantly from roundstart at a cost of 30 sheets of metal (1/5th of science's roundstart allowance).
A comparable situation would robotics being able to print pre-calculated cans of burn mix for Toxins.

**Comment by AlexCrimson** [comment]
Compared to what? Taking the extra few minutes to print off items from the Protolathe and decon those instead? You can do R&D extremely fast either way, and eventually you reach a point where you need mining materials to get and build the good stuff.

I agree its silly how fast you can do R&D, but changing one item wont really affect much.

**Comment by Dorsisdwarf** [comment]
I'm not doing a total R&D rebalance here. I'm removing a ridiculously 'overpowered' element of the system to help slow down R&D. Every second they aren't packing more guns than security is a good second.

**Comment by AlexCrimson** [comment]
Well fine then. I dont think this is going to change anything, but you seem to be convinced otherwise.

**Comment by Dorsisdwarf** [comment]
Not all changes have to be gamechanging earthshattering refactors, Alex.

**Comment by AlexanderUlanH** [comment]
Why can't we just increase the research requirements so you actually have to get some of the requisite tech before you can make it?  It's hardly an underpowered piece of mech equipment.

**Comment by Dorsisdwarf** [comment]
Because it's standard equipment for a standard mech (that you can order from cargo)?

**Comment by palpatine213** [comment]
This has a side effect of nerfing the availability of drones. Before, the mech syringe guns easily provided the required tech needed to build them. I'd recommend lowering drone biotech requirements to 3 to keep things balanced.

**Comment by Dorsisdwarf** [comment]
Just get a brain or two from genetics monkies and decon them.

**Comment by Lumipharon** [comment]
This is pretty dumb.

Firstly, the old method of triple syringe guns is SLOWER then other methods that involve never leaving R&D - you only need 1, to get bio 5, which otherwise requires a slime core from xenobio.

Secondly, as shown through it's infinite chem generation of all sorts of nasty chems, it IS a very strong weapon.


Make it actually require some research to build rather then dunking it's origin tech - also you can order oddy boards from cargo, not its equipment.

**Comment by Iamgoofball** [comment]
:-1:

i like it as-is

**Comment by kevinz000** [comment]
:-1: MUH RESEARCH

**Comment by babinsquared** [comment]
This just slightly modifies the cookbook.

**Comment by SconesC** [comment]
R&D will always be a fetch quest, no reason to make it a slightly more tedious fetch quest

**Comment by Dorsisdwarf** [comment]
So your proposed solution to 'R&D is a shit system' is 'Don't try and tweak the system'?

**Comment by phil235** [comment]
conflict is with goon chemistry reorganization.

**Comment by Dorsisdwarf** [comment]
Gurrrrr

**Comment by AlexanderUlanH** [comment]
:+1: as long as R&D still has a level 4 electromagnetic theory item.

**Comment by Dorsisdwarf** [comment]
They don't, which is why this just became a WIP. Thank you to the lividly furious people in OOC for screaming this fact at me.

**Comment by AlexCrimson** [comment]
As i said, all the Mech Syringe Gun tech levels can be obtained with other items. Even Bio 5(Slime Core is Bio 4, so it will get you Bio 5). The only issue is EM 4. The only other items i have found with EM 4 are the Advanced Mass Spectrometer and one of the stock part items(High-Powered Micro Laser i think?). Neither of them have the reliability to be Deconned in the Analyzer until you fully upgrade it.

Like i said in OOC, just increase the reliability of the Advanced Mass Spectrometer and decrease the reliability of the Mech Syringe Gun. That solves all the problems, no?

Id post the R&D guide i use, but you would probably just nerf it. So i wont.

**Comment by Dorsisdwarf** [comment]
Well, as it is, you can decon the Advanced Mass spectro without advanced parts, just use it over and over until it breaks. I'm currently making the spectrometers not malfunction though, so they provide a reliable tech path.

File: code/modules/reagents/Chemistry-Goon-Other.dm [hunk 1]
```diff
@@ -283,4 +283,4 @@ datum/reagent/concentrated_barbers_aid/reaction_mob(var/mob/living/M, var/volume
 		H.facial_hair_style = "Very Long Beard"
 		H.update_hair()
 	..()
-	return
\ No newline at end of file
+	return
```
Comment (Dorsisdwarf) [inline code/modules/reagents/Chemistry-Goon-Other.dm]: Why is this a chnage?!?

**Comment by AlexCrimson** [comment]
Slime cores are neat, but you might want to run that by the coder who added implants before you merge this. If i remember right he nerfed some stuff that gave bio 6 specifically to stop R&D getting it so easily. As it unlocks the OP implants.

**Comment by Dorsisdwarf** [comment]
Oh shit, I forgot about implants. I thought they were nerfed to bio 10?

File: code/modules/reagents/Chemistry-Goon-Other.dm (hunk 1 above)
Comment (phil235) [inline code/modules/reagents/Chemistry-Goon-Other.dm]: No idea, it happens often but it's harmless.

**Comment by AlexCrimson** [comment]
Err im not sure about that. Some of them may be bio 10, but im sure you could unlock a bunch of them at bio 6, which required deconning a Xeno brain.

**Comment by Miauw62** [comment]
Still merge conflicts.
Please rebase instead of merging (for much the same reasons as using the map merger, e.g. it keeps our history tidy and small-ish.)

**Comment by Dorsisdwarf** [comment]
I still don't know how to rebase.

File: code/modules/reagents/Chemistry-Goon-Other.dm (hunk 1 above)
Comment (Dorsisdwarf) [inline code/modules/reagents/Chemistry-Goon-Other.dm]: So what IS causing the merge conflicts?

**Comment by palpatine213** [comment]
@Dorsisdwarf What the hell are you doing with implants? The biotech 7 requirement is going through in the next update. Also, why are you raising the cns rebooter implant to bio 7? That's only going to be for the xray implant.

**Comment by Dorsisdwarf** [comment]
Because I made top tier slime cores bio 5, which would have let people get xrays with reliable regularity, I buffed everything which was Bio 6 (alium brains only) to Bio 7 (also alium brains only)

**Comment by palpatine213** [comment]
Alright. the issue is, that currently the only way to get the top tier implants is with xeno brains, OR the traitor microlaser. with that change, your effectively nerfing the lasers use in research

**Comment by Dorsisdwarf** [comment]
Traitor microlaser is bio 6, right? Just use it after maxing with high-tier slime cores

**Comment by palpatine213** [comment]
bio 5

**Comment by Lumipharon** [comment]
Still don't see why it's not easier to just give the syringe gun some required tech, to avoid this 'issue' of triple syringe gun research.

**Comment by SconesC** [comment]
I still don't understand the motivation behind making this change

If you want to change R&D you should just overhaul it instead of trying to make the fetch quest even more tedious

**Comment by Dorsisdwarf** [comment]
Because as it is, there was a single element that made basically three whole branches of research irrelevant? 
Because this means you have to spend some actual effort to max everything?
Because this encourages you to do xenobio so you can get maximum biotech?
Because this ends the endless tide of 'Why do abductor items have no research levels coders a shit'?
Because R&D should require more than 3 of the same roundstart printable cheap item to fill in nearly half of all possible research levels?

**Comment by Dorsisdwarf** [comment]
Also, a complete overhaul to origin techs is unfeasible, because you'd need a damn code freeze where nobody could touch anything except origin techs in order to avoid it conflicting with EVERYTHING

**Comment by Lumipharon** [comment]
Then just give the syringe gun required tech levels. Then you can still use it to reach the limit of what it's origin tech can get you, but you'd have to do some research which would for example fill in what otherwise gets done by the first 2 syringe guns.

'encouraging you to do xenobio' is literally 'kill one slime at round start for bio 5 instead of a syringe gun and never come back.

**Comment by Dorsisdwarf** [comment]
Or just remove it's comically high tech levels and buff something else?
Your suggestion would make them even less used for intended function, IE: Oddys, because robo can't kit out their mechs any more because it's not a syringe gun, it's a datapacket for science.

**Comment by Lumipharon** [comment]
Because it's the least complicated solution to the issue that you've raised.
http://pastebin.com/xhYDr61w is my method, it uses one syringe gun, because when I used to do the shitty old x3 syringe gun method it was SLOWER.

Saying syringe guns won't get used on an oddy is the same as people ordering oddy boards from cargo because they can't make them at round start. Sure it is minimal effort, but you can literally get research high enough to make them then print them yourself (as roboticists have full access to R&D these days) faster then it would take to get them from cargo.

Not sure what you would arbitrarily buff in terms of origin tech to make up for the syringe gun change, other then specific slime cores, which is awful because xenobio is slow and terrible, and relies on heavy RNG and time gated evolutions.

**Comment by AlexCrimson** [comment]
Screw you Lumipharon, dont post that shit in a PR with a coder who is intent on nerfing the speed of R&D. Are you nuts?

**Comment by chimersable** [comment]
RnD on my server can't be complicated because rounds are too short to invest a lot of time in research. Also, only newbs try to deconstruct 3 syringe guns there are much better, faster, more productive ways to get that research and then use those guns to cap it off.

 Btw, dorsis isn't publishing this add on because he wants to make the game more fun to do Rnd, according to him in an earlier post it's to slow down science's ability to make guns.

**Comment by Dorsisdwarf** [comment]
Slow down science in general is a point. But as you said, it's not really slowing down people who know what they're doing, which brings it to the other point, which is that it's stupid to have such a cheap roundstart item have a techline of 3/4/4/3

**Comment by AlexCrimson** [comment]
How can you slow down getting guns when the syringe gun literally has no effect on R&D getting guns? Those R&D guns require mining ores.

**Comment by Dorsisdwarf** [comment]
Whydo you insist, on the forums, and now on the github, continuously misrepresenting my posts over and over and over again so you can make some kind of a point? Why are you so dead-set against this change?
 I really, honestly cannot comprehend why if it has no effect, you keep complaining that it will ruin science, but that if someone insinuates it will change science, you do a 180 and declare loudly that it won't change anything nothing will change the change is pointless.

**Comment by chimersable** [comment]
"But as you said, it's not really slowing down people who know what they're doing" 
So you want to make RnD more confusing for the casual or new player?

I play on yogs which seems to have the gun locked until the correct research is met, which is a much better solution to changing the values of the research, anyways. You would have my support in bug posting this which is what it seems like.

**Comment by Dorsisdwarf** [comment]
Yogstation runs a /tg/ clone, don't they? They certainly don't seem to have their own Repo that I can find.

**Comment by Iamgoofball** [comment]
Dorisdwarf they give code upon request, just flag down a dev.

**Comment by Dorsisdwarf** [comment]
Allahu snackbar, merge conflicts ho

**Comment by Dorsisdwarf** [comment]
I CBA to fix this

**Pull request closed without merging**